### PR TITLE
FAQ - Change exposure log paths and remove deprecated info

### LINKED ISSUE #618: FAQ entry: path for exposure checks needs to be corrected
## Where to find the issue
https://www.coronawarn.app/de/faq/#exposure_check
https://www.coronawarn.app/en/faq/#exposure_check

## Describe the issue
The path to check the risk notificationsis more hidden and require one extra tap. 

## Suggested change
Pls. correct the path as follows:

**EN**
Settings - 'Exposure Notifications' - 'Exposure Logging Status' - 'Exposure Checks' 

**DE**
Einstellungen - 'Begegnungsmitteilungen' - 'Status von Begegnungsaufzeichnungen' - Begegnungsüberprüfungen

### PR DESCRIPTION
**Pull request by heinezen**
Fixes #618 

Incorporates additional info for background updates since v1.7.0

### CODE REVIEW

File: src/data/faq.json [hunk 1]
```diff
@@ -199,7 +199,12 @@
...
+                            "When the app was running constantly during the last two weeks, you should see at least one check every day for all of the past 14 days.",
+                            "Please note: The log is just the history. After every background update, the exposures of the last 14 days are checked and the risk status calculated. The important thing is that the exposure log shows at least one check for the last 24 hours.",
+                            "It might be that entries are displayed for less than 14 days, although exposure was checked correctly. There are several reasons:",
+                            "<ol><li>The log has a fixed limit of 100 entries on iOS devices. If there are more entries, only the latest are kept. We're in contact with Apple so that the complete log will be displayed. The log limit of 100 entries on Android devices has been recently removed by Google at the end of August 2020.</li><li>If you use the app only recently (let's say for 5 days), only checks that were made after setting up the app are displayed - at least one for every day that the Corona-Warn-App was running constantly.</li></ol>"
```
Comment (MikeMcC399) [inline src/data/faq.json:207]: Suggest changing:

The log limit of 100 entries on Android devices has been recently removed by Google at the end of August 2020.

to:

Android devices are limited to 300 entries.

File: src/data/faq_de.json [hunk 2]
```diff
@@ -199,7 +199,12 @@
...
+                            "Es kann vorkommen, dass hier Einträge für weniger als 14 Tage angezeigt werden, obwohl die Risiko-Ermittlung an jedem der letzten 14 Tagen korrekt durchgeführt wurde. Dafür gibt es mehrere Gründe:",
+                            "<ol><li>Diese Übersicht hat ein festes Limit von 100 Einträgen auf iOS Geräten. Wird dieses Limit überschritten, werden nur die neuesten Einträge behalten. Sie sehen dann typischerweise die Einträge von 7 Tagen. Wir sind mit Apple in Kontakt, damit die Übersicht in Zukunft vollständig angezeigt wird. Die Protokollgrenze von 100 Einträgen auf Android-Geräten wurde kürzlich von Google Ende August 2020 entfernt.</li><li>Wenn Sie die App erst seit Kurzem nutzen (zum Beispiel seit 5 Tagen), werden entsprechend nur Überprüfungen nach der Installation der App angezeigt - mindestens eine für jeden Tag, an dem die Corona-Warn-App lief.</li></ol>"
```
Comment (MikeMcC399) [inline src/data/faq_de.json:207]: Suggest changing:

Die Protokollgrenze von 100 Einträgen auf Android-Geräten wurde kürzlich von Google Ende August 2020 entfernt.

to

Android-Geräten haben eine Protokollgrenze von 300 Einträge.

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
The current text implies that the limit of 100 was removed and there is now no limit. Android devices are limited to 300 entries. The log is not allowed to increase without limit.

See my note in https://github.com/corona-warn-app/cwa-app-android/issues/1020#issuecomment-722373534 and the exposure log linked to https://github.com/corona-warn-app/cwa-app-android/issues/1817#issuecomment-739553078

Also the text mentioning August 2020 as being recent is no longer up-to-date. You could possibly say that the limit was increased from 100 to 300 in August, but that is a historical fact that is hardly relevant today. The important point is the current limit of 300.

**Comment by heinezen** [comment]
@MikeMcC399 Thanks!

I've updated the answer again. Only used a slightly different phrasing and completely removed the mentioning of August 2020.

**Comment by MikeMcC399** [comment]
@heinezen 
> I've updated the answer again. Only used a slightly different phrasing and completely removed the mentioning of August 2020.

The limit of 300 for Android is sufficient for normal use. (14 days x 6 times per day = 84 entries, which gives a reserve of 216 entries.) I don't know a reason why you would want to ask Google to increase the limit above 300. There is a danger of filling up device memory if there is a problem. We've seen some error situations where the limit of 300 is hit because the aging out of entries older than 14 days is not working.

For iOS the statement that you are in contact with Apple has been there a long time. It doesn't sound like anything is going to get resolved. I'm not involved with iOS so I don't have any further insight there.

**Comment by heinezen** [comment]
@MikeMcC399 The last information I found on this is that we are still trying to get all OSs to display the full logs for error reporting. For iOS, this might be resolved in a later ENA version.

I've removed the sentence now, since the log shouldn't fill up under normal circumstances.

----
Corona-Warn-App Open Source Team

File: src/data/faq_de.json [hunk 3]
```diff
@@ -199,7 +199,12 @@
...
+                            "Bitte beachten Sie: Diese Übersicht zeigt nur die Historie an. Nach jeder Hintergrundaktualisierung werden die Begegnungen der letzten 14 Tage geprüft und das Risiko daraus ermittelt. Wichtig ist also, dass die Übersicht mindestens eine Begegnungsüberprüfung während der letzten 24 Stunden anzeigt.",
+                            "Es kann vorkommen, dass hier Einträge für weniger als 14 Tage angezeigt werden, obwohl die Risiko-Ermittlung an jedem der letzten 14 Tagen korrekt durchgeführt wurde. Dafür gibt es mehrere Gründe:",
+                            "<ol><li>Diese Übersicht hat ein festes Limit von 100 Einträgen auf iOS Geräten. Auf Android-Geräten beträgt das Limit 300 Einträge. Wird dieses Limit überschritten, werden nur die neuesten Einträge behalten. Sie sehen dann typischerweise die Einträge von 7 Tagen.</li><li>Wenn Sie die App erst seit Kurzem nutzen (zum Beispiel seit 5 Tagen), werden entsprechend nur Überprüfungen nach der Installation der App angezeigt - mindestens eine für jeden Tag, an dem die Corona-Warn-App lief.</li></ol>"
```
Comment (MikeMcC399) [inline src/data/faq_de.json:207]: "Diese Übersicht hat ein festes Limit von 100 Einträgen auf iOS Geräten. Auf Android-Geräten beträgt das Limit 300 Einträge. Wird dieses Limit überschritten, werden nur die neuesten Einträge behalten. **Sie sehen dann typischerweise die Einträge von 7 Tagen.**"

On Android it is not expected that the limit would be reached and typically you see entries for 16 days. I would just remove the sentence "Sie sehen dann typischerweise die Einträge von 7 Tagen."

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
Thank you for removing the comment about trying to get the limit reduced.

I noticed one other point in the German text "Sie sehen dann typischerweise die Einträge von 7 Tagen." which is not true for Android and the equivalent statement isn't present in the English text. Please consider removing this statement.

**Comment by dsarkar** [comment]
image

image


@MikeMcC399 @loki-cui - Please review. Thanks

**Review by dsarkar — APPROVED** [review]

**Review by dsarkar — APPROVED** [review]

**Review by loki-cui — APPROVED** [review]

**Review by dsarkar — APPROVED** [review]

**Comment by MikeMcC399** [comment]
@dsarkar 
LGTM. Thank you!

**Pull request merged**